welcome/index.html

```diff
@@ -0,0 +1,242 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+    <title>A heroku-like platform for IoT projects | Experimental Platform</title>
+    <link rel="stylesheet" href="experimental.platform.min.css">
+    <meta name="author" content="Protonet GmbH, Hamburg, Germany">
+    <meta name="description" content="Experimental Platform provides an easy way to deploy apps on local hardware without the hassle to setup the environment. It enables developers and makers to focus on the things that matter.">
+    <meta content="Experimental Platform" property="og:site_name">
+    <meta content="website" property="og:type">
+    <meta content="https://experimental-platform.github.io/img/og_image.jpg" property="og:image">
+    <meta content="Experimental Platform - Heroku-like platform for IoT projects" property="og:title">
+    <meta content="https://experimental-platform.github.io/" property="og:url">
+    <meta content="Experimental Platform provides an easy way to deploy apps on local hardware without the hassle to setup the environment. It enables developers and makers to focus on the things that matter." property="og:description">
```

Comment:
don't we need a ``<meta name="viewport" content="width=device-width, initial-scale=1">``?